VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/tasks-table/tasks-table.component.html

```diff
@@ -1,42 +1,48 @@
 <div class="task-header">
   <span class="task-header-body govuk-heading-s">Alerts for this hearing</span>
 </div>
-<div id="tasks-list" style="overflow: auto; height: xyz;" class="task-list">
+<div id="tasks-list" style="overflow: auto; height: xyz" class="task-list">
```

Comment:
move this inline style to the stylesheet & remove the height property

Reply:
I can remove it but that was already there

Reply:
inline styles added to css file
height: xyz removed